lib/runtime/addon.js

```diff
@@ -341,4 +340,34 @@ export default class Addon {
     return configModule && (configModule.default || configModule);
   }
 
+  get blueprintNames() {
+    let names = fs.readdirSync(path.join(this.dir, 'blueprints'));
+    return names;
```

Comment:
Sorry, should have been clearer - I meant to suggest that we get rid of this method and consolidate it into the blueprints getter.